frontends/mit-open/src/AppProviders.test.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react"
+
+import { urls, setMockResponse } from "api/test-utils"
+import { learningResources as factory } from "api/test-utils/factories"
+
+import { renderTestApp, screen } from "./test-utils"
+
+jest.mock("posthog-js/react", () => ({
+  PostHogProvider: (props: { children: React.ReactNode }) => (
+    <div data-testid="phProvider">{props.children}</div>
+  ),
+}))
+
+describe("PostHogProvider", () => {
+  it("Renders with PostHog support if enabled", async () => {
+    window.SETTINGS = {
+      ...window.SETTINGS,
+      posthog: {
+        api_key: "", // pragma: allowlist secret
+        enabled: true,
+      },
+    }
+
+    const resources = factory.resources({ count: 4 })
+    setMockResponse.get(urls.learningResources.list(), resources)
```

Comment:
`renderTestApp` renders the whole app, which makes some API calls, you you had to mock, which is a pain.

I would recommend using `renderWithProviders` for this:
```
renderWithProviders(<div data-testid="some-children" />)
```
then you won't need to mock the API calls.